src/components/private-route/private-route.tsx

```diff
@@ -0,0 +1,19 @@
+import {Navigate} from 'react-router-dom';
+import { AppRoute, AuthorizationStatus } from '../../const';
+
+type PrivateRouteProps = {
+  authorizationStatus: AuthorizationStatus;
+  children: JSX.Element;
+}
+
+function PrivateRoute(props: PrivateRouteProps): JSX.Element {
```

Comment:
Лучше деструктуризацию пропсов сразу делать 
```suggestion
function PrivateRoute({authorizationStatus, children}: PrivateRouteProps): JSX.Element {
```